config/secrets.yml

```diff
@@ -12,11 +12,14 @@
 
 development:
   secret_key_base: 777689ac6ca9922e06ac2968095accdc5569089383869512ff7c7c9d81dbe940ae2104346332f91d88b2cdde0352d8622d59ee7329499101824fc5718759a877
+  link_checker_api_secret_token: evpsuoPKMc2kU9MkKZzFuF7b
 
 test:
   secret_key_base: 70ebfd8ab10715f055bea44e49057544e6d232e374ba191b54a3f490d86cc8a5327d329d8cb6014b3968ade5efacc48563ec16d85d2305ee72a973307e130506
+  link_checker_api_secret_token: ENGgg9TEmRRxZg9FEAC6qrVu
 
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
 production:
   secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  link_checker_api_secret_token: <%= ENV["LINK_CHECKER_API_SECRET_TOKEN"] %>
```

Comment:
We'll need to add this to [`govuk-puppet`](https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk/manifests/apps/whitehall.pp#L128) to make sure it gets set.